ruby/exercises/promedio_array.rb

```diff
@@ -11,3 +11,32 @@
 #     Input: [1, 5, 87, 45, 8, 8] Resultado: 25
 #     Input: [2,5,13,20,16,16,10] Resultado: 11
 #     Input: [1,2,15,15,17,11,12,17,17,14,13,15,6,11,8,7] Resultado: 11
+
+def promedio_e (calificaciones)
+    #con este si me da los resultados 2, 25, 11 y 11 --> porque la division la deja en entero
+    (calificaciones.sum / calificaciones.count).round
+
+end
+
+def promedio_f (calificaciones)
```

Comment:
al declarar los metodos en ruby por lo general no dejamops un espacio entre el nombre del metodo y los parentesis
```suggestion
def promedio_f(calificaciones)
```